rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+
+#### 21 Notice to member
+
+1. Before disciplinary action is taken against a member, the Secretary must give written notice to the member—
+  1. satating that the Association proposes to take disciplinary action against the member; and
+  2. stating the grounds for the proposed disciplinary action; and
+  3. specifying the date, place and time of the meeting at which the Committee intends to consider the disciplinary action (the disciplinary meeting); and
+  4. advising the member that he or she may do one or both of the following—
+    1. attend the disciplinary meeting and address the Committee at that meeting;
+    2. give a written statement to the Committee at any time before the disciplinary meeting; and 
+  5. setting out the member's appeal rights under rule 23.
+
+2. The member must be given reasonable notice of the meeting.
+
+#### 22 Decision of subcommittee
+
+1. At the disciplinary meeting, the Committee must—
+  1. give the member an opportunity to be heard; and
+  2. consider any written statement submitted by the member.
+
+2. After complying with subrule 1, the Comittee may—
+  1. take no further action against the member; or
+  2. subject to subrule 3—
+    1. reprimand the member; or
+    2. suspend the membership rights of the member for a specified period; or
+    3. suspend or expel the member from attendance at an event operated by the Association
+    4. expel the member from the Association.
+
+3. The Committee may not fine the member.
+
+4. The suspension of membership rights or the expulsion of a member by the Committee under this rule takes effect immediately after the vote is passed.
+
+#### 23 Appeal rights
+
+1. A person whose membership rights have been suspended or who has been expelled from the Association under rule 22 may give notice to the effect that he or she wishes to appeal against the suspension or expulsion.
+
+2. The notice must be in writing and given—
+  1. to the disciplinary subcommittee immediately after the vote to suspend or expel the person is taken; or
+  2. to the Secretary not later than 48 hours after the vote.
+
+3. If a person has given notice under subrule 2, a disciplinary appeal meeting must be convened by the Committee as soon as practicable, but in any event not later than 21 days, after the notice is received.
+
+4. Notice of the disciplinary appeal meeting must be given to each member of the Association who is entitled to vote as soon as practicable and must—
+  1. specify the date, time and place of the meeting; and
+  2. state—
+    1. the name of the person against whom the disciplinary action has been taken; and
+    2.  the grounds for taking that action; and
+    3. that at the disciplinary appeal meeting the members present must vote on whether the decision to suspend or expel the person should be upheld or revoked.
+
+### Division 3—Grievance procedure
+
+#### 25 Application
```

Comment:
This rule was removed because it isn't applicable to this organisation. This Constitution is based on the ["Model Rules for an Incorporated Association"](https://www.consumer.vic.gov.au/library/forms/clubs-and-not-for-profits/incorporated-associations/model-rules-for-an-incorporated-association.doc).